docs/src/pages/components/authenticator/advanced/current-route.angular.mdx

```diff
@@ -26,3 +26,23 @@ You can use `AuthenticatorService` to access `route` string that represents the
   <amplify-authenticator></amplify-authenticator>
 </ng-container>
 ```
+
+#### Authentication Check
+
+If you just need to check if you're authenticated or not, you can use the more straight forward `AuthenticatorService` to access the `authCheck` string. The `authCheck` string can represent the following states:
```

Comment:
I love the docs here! My only nit here is naming -- have you considered any alternatives to `authCheck`? `authState` and `authStatus` come to my mind. 

IMO `authCheck` sounds like a function, but this is obviously subjective and am open to suggestions.

Reply:
Ok, I like `authStatus`, I can change it to that

Reply:
I'm bad at naming :(

Reply:
dd3e3f0d9511a8d667d8223306538db8f0a84f96